templates/workspace_detail.html

```diff
@@ -270,6 +270,21 @@ <h1 class="mb-2 text-3xl tracking-tight break-words font-bold text-slate-900 sm:
         {% /card %}
       {% endif %}
 
+      {% #card title="Reports" %}
+        {% #list_group small=True %}
+          {% for report in reports %}
+          {% #list_group_item href=report.get_absolute_url %}
+            {{ report.title }}
+            {% if report.is_draft %}
+              {% pill text="Draft" variant="primary" %}
```

Comment:
Could we change this to "Not published" rather than "Draft" please?

Draft suggests that it could change. It also simplifies the terminology by saying that a report is either published or not. And, since "Not published" uses a negative, it feels like something you would want to change (by publishing!).